pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -449,8 +464,13 @@ func (c *clusterAutoscaler) computeCommand() []string {
 	}
 	gardencorev1beta1.SetDefaults_ClusterAutoscaler(c.config)
 
+	expanderMode := *c.config.Expander
+	if workersHavePriorityConfigured {
+		expanderMode = ensureExpanderInExpanderConfig("priority", expanderMode)
```

Comment:
```suggestion
		expanderMode = ensureExpanderInExpanderConfig(gardencorev1beta1.ClusterAutoscalerExpanderPriority, expanderMode)
```

Let's use this also here.